plugins/destination/plugin_testing.go

```diff
@@ -170,40 +197,10 @@ func PluginTestSuiteRunner(t *testing.T, newPlugin NewPluginFunc, destSpec specs
 	})
 }
 
-func createTestResources(table *schema.Table, sourceName string, syncTime time.Time, count int) []schema.DestinationResource {
-	resources := make([]schema.DestinationResource, count)
-	for i := 0; i < count; i++ {
-		resource := schema.DestinationResource{
-			TableName: table.Name,
-			Data:      testdata.GenTestData(table),
-		}
-		_ = resource.Data[0].Set(sourceName)
-		_ = resource.Data[1].Set(syncTime)
-		resources[i] = resource
-	}
-	return resources
-}
-
-func sortResources(table *schema.Table, resources []schema.DestinationResource) {
-	cqIDIndex := table.Columns.Index(schema.CqIDColumn.Name)
-	syncTimeIndex := table.Columns.Index(schema.CqSyncTimeColumn.Name)
-	sort.Slice(resources, func(i, j int) bool {
-		// sort by sync time, then UUID
-		if !resources[i].Data[syncTimeIndex].Equal(resources[j].Data[syncTimeIndex]) {
-			return resources[i].Data[syncTimeIndex].Get().(time.Time).Before(resources[j].Data[syncTimeIndex].Get().(time.Time))
-		}
-		return resources[i].Data[cqIDIndex].String() < resources[j].Data[cqIDIndex].String()
-	})
-}
-
-func sortCQTypes(table *schema.Table, resources []schema.CQTypes) {
-	cqIDIndex := table.Columns.Index(schema.CqIDColumn.Name)
+func sortRecordsBySyncTime(table *schema.Table, records []arrow.Record) {
```

Comment:
This assumes that the records each only contain a single row, right? I guess it's fine for testing